src/core/namespace.c

```diff
@@ -319,8 +327,11 @@ static int apply_mount(
         r = mount(what, m->path, NULL, MS_BIND|MS_REC, NULL);
         if (r >= 0)
                 log_debug("Successfully mounted %s to %s", what, m->path);
-        else if (m->ignore && errno == ENOENT)
-                return 0;
+        else {
+                if (m->ignore && errno == ENOENT)
+                        return 0;
+                log_error_errno(errno, "Failed mounting %s to %s: %s", what, m->path, strerror(errno));
```

Comment:
Actually, all `log_*` don't work. See https://github.com/systemd/systemd/blob/61233823aa4b0fe9605e0a7cd77261b3c5bca8e9/src/core/execute.c#L1614
```c

        log_forget_fds();

        r = close_remaining_fds(params, runtime, socket_fd, fds, n_fds);
```
We should `log_open`, `log_*`, `log_close`. Similar to https://github.com/systemd/systemd/blob/61233823aa4b0fe9605e0a7cd77261b3c5bca8e9/src/core/execute.c#L1714